interface/modules/custom_modules/oe-module-comlink-telehealth/src/Bootstrap.php

```diff
@@ -0,0 +1,346 @@
+<?php
+
+/**
+ * This bootstrap file connects the module to the OpenEMR system hooking to the API, api scopes, and event notifications
+ *
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2022 Comlink Inc <https://comlinkinc.com/>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace Comlink\OpenEMR\Modules\TeleHealthModule;
+
+use Comlink\OpenEMR\Modules\TeleHealthModule\Controller\Admin\TeleHealthPatientAdminController;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Controller\Admin\TeleHealthUserAdminController;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Controller\TeleconferenceRoomController;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Controller\TeleHealthCalendarController;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Controller\TeleHealthPatientPortalController;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Controller\TeleHealthVideoRegistrationController;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Repository\CalendarEventCategoryRepository;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Repository\TeleHealthPersonSettingsRepository;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Repository\TeleHealthProviderRepository;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Repository\TeleHealthSessionRepository;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Repository\TeleHealthUserRepository;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Services\TelehealthRegistrationCodeService;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Services\TeleHealthRemoteRegistrationService;
+use OpenEMR\Common\Logging\SystemLogger;
+use OpenEMR\Common\Twig\TwigContainer;
+use OpenEMR\Core\Kernel;
+use OpenEMR\Events\Appointments\AppointmentSetEvent;
+use OpenEMR\Events\Core\TwigEnvironmentEvent;
+use OpenEMR\Events\Globals\GlobalsInitializedEvent;
+use OpenEMR\Events\Main\Tabs\RenderEvent;
+use OpenEMR\Services\Globals\GlobalSetting;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+class Bootstrap
+{
+    const OPENEMR_GLOBALS_LOCATION = "../../../../globals.php";
+    const MODULE_INSTALLATION_PATH = "/interface/modules/custom_modules/";
```

Comment:
just making sure no webroot issues here :) (ie. on demos where it uses openemr)
as an aside, there is a openemr-cmd command in the easy dev docker that will stick a webroot in for testing (since we all have been developing without any webroot for awhile and only catch these bugs when they get to the demos)
![image](https://user-images.githubusercontent.com/278968/197964524-07f147b5-dd76-41ab-aa6b-7cd997f8510f.png)

Reply:
See lines 223 and 243 of Bootstrap.php.  I make sure the webroot is populated when I use the constant.